lib/hooks/queries/court/useCourtCases.ts

```diff
@@ -17,7 +18,7 @@ export const useCourtCases = () => {
   const enabled = !!sdk && isRpcSdk(sdk);
 
   const query = useQuery<CourtCaseInfo[]>(
-    [id, courtCasesRootKey],
+    [id, courtCaseRootKey, "all"],
```

Comment:
Is the "all" value needed? If a single court case is invalidated shouldn't that invalidate the whole list?